# Request
sensor_msgs/Image sdf_map
---
# Response
sensor_msgs/PointCloud keypoints